packages/amplify_api_plugin_interface/lib/src/RestAPI/RestResponse.dart

```diff
@@ -13,9 +13,37 @@
  * permissions and limitations under the License.
  */
 
+import 'dart:convert';
 import 'dart:typed_data';
 
+/// An HTTP response from a REST API.
 class RestResponse {
-  Uint8List data;
-  RestResponse({required this.data});
+  /// The response status code.
+  final int statusCode;
```

Comment:
question: IIRC the status code is only available for error on iOS. PR description mentions adding stuff when available from platform. Am I correct in understanding the status code here is only available on IOS? Are there other platform discrepancies here?

Reply:
Yeah, you're right - my mistake. The status code is not available on iOS. I had put "200" in for success cases but that's technically incorrect.